src/databricks/labs/ucx/install.py

```diff
@@ -746,37 +791,31 @@ def _job_wheel_task(self, jobs_task: jobs.Task, task: Task, remote_wheel: str) -
 
     def _job_clusters(self, names: set[str]):
         clusters = []
-        spark_conf = {
-            "spark.databricks.cluster.profile": "singleNode",
-            "spark.master": "local[*]",
-        }
-        if self._config.spark_conf is not None:
-            spark_conf = spark_conf | self._config.spark_conf
-        spec = compute.ClusterSpec(
-            data_security_mode=compute.DataSecurityMode.LEGACY_SINGLE_USER,
-            spark_conf=spark_conf,
-            custom_tags={"ResourceClass": "SingleNode"},
-            num_workers=0,
-            policy_id=self.config.policy_id,
-        )
+        cluster_specs = self._config.cluster_specs
```

Comment:
No, this gets too fragile. Installer is the critical component and this introduces severe risk of breakage. Revert.